### PR TITLE
Add tool for updating configs

### PR DESCRIPTION
**Pull request by lholznagel**
The PR adds a script wich updates the config files of a project.

Please take a look at the code. 
I think the easiest thing is to add a :+1: if you think this can be merged and you are satisfied with the code :)

### CODE REVIEW

**Comment by remojansen** [comment]
Kudos for this! Really awesome work :+1:

**Comment by lholznagel** [comment]
@otbe can you take a final look, so that I can merge it tomorrow?

File: app/update.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+import Utils from './utils';
+
+export default class Update {
+  private branch: string;
+  private commit: string;
+  private configRepo: string;
+  private currentBranch: string;
+
+  constructor(flags: any) {
```
Comment (otbe) [inline app/update.ts]: Maybe you can change this ```any``` to something like
```
interface {
  branch?: string;
  commit?: string;
  configRepo?: string;
}
```

File: app/update.ts [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        title: 'Pushing Branch',
+        task: () => execa('git', ['push', 'origin', this.branch])
+      }, {
+        title: 'Cleanup time',
+        task: () => this.cleanUp()
+      }
+    ])
+      .run()
+      .catch((error) => {
+        console.log('There was a error :(');
```
Comment (otbe) [inline app/update.ts]: Typo ```an error```

File: app/update.ts [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+        Utils.writeError(error);
```
Comment (otbe) [inline app/update.ts]: Its not a big problem, but ```Utils.writeError(error);``` is async. So in some circumstances it is possible that the error file is not (complete) written to the disk but the code proceeds execution.

**Pull request merged**